controllers/release/adapter.go

```diff
@@ -256,13 +257,25 @@ func (a *adapter) EnsureReleaseIsProcessed() (controller.OperationResult, error)
 		return controller.RequeueWithError(err)
 	}
 
+	// Don't check for error as it is expected if no RoleBinding exists yet
+	roleBinding, _ := a.loader.GetRoleBindingFromReleaseStatus(a.ctx, a.client, a.release)
```

Comment:
Then use `errors.IsNotFound(err)`.

Reply:
That won't be the error type if the rolebinding field in the release status is empty